data/src/main/java/com/microsoft/azure/kusto/data/exceptions/OneApiError.java

```diff
@@ -15,6 +20,12 @@ public OneApiError(String code, String message, String description, String type,
         this.permanent = permanent;
     }
 
+    public static OneApiError[] fromJsonArray(ArrayNode array) {
+        return StreamSupport.stream(Spliterators.spliteratorUnknownSize(array.elements(), Spliterator.ORDERED), false)
```

Comment:
why all this splitator things ? Stream . not good enough?

Reply:
I think I checked and that's the "correct" way to to it.

Reply:
If you have another direction let me know